src/ctl_int.rs

```diff
@@ -43,6 +67,11 @@ impl Ctl {
     pub fn card_info(&self) -> Result<CardInfo> { CardInfo::new().and_then(|c|
         acheck!(snd_ctl_card_info(self.0, c.0)).map(|_| c)) }
 
+    /// Wrapper around [snd_ctl_pcm_next_device](https://www.alsa-project.org/alsa-doc/alsa-lib/control_8c.html#accbb0be6e5ca7361ffec0ea304ed1b05).
+    pub fn pcm_next_device(&self) -> DeviceIter {
```

Comment:
This function can be removed, `DeviceIter::new` can be public, and the comment moved to the DeviceIter struct. Follows conventions in this crate better